upup/models/config/components/kube-apiserver/_do/kube-apiserver.do.options

```diff
@@ -0,0 +1,2 @@
+KubeAPIServer:
+  CloudProvider: external
```

Comment:
Created an issue to clean this up for all clouds https://github.com/kubernetes/kops/issues/3253